packages/admin/admin/src/form/file/FileSelect.tsx

```diff
@@ -134,7 +154,7 @@ export const FileSelect = <AdditionalValidFileValues = Record<string, unknown>,>
                     </ErrorMessage>
                 </Error>
             )}
-            {Boolean(filesInfoText) && <FilesInfoText {...slotProps?.filesInfoText}>{filesInfoText}</FilesInfoText>}
+            {Boolean(filesInfoText && !readOnly) && <FilesInfoText {...slotProps?.filesInfoText}>{filesInfoText}</FilesInfoText>}
```

Comment:
Why shouln't it not be shown when the select is read-only?

Reply:
This is only information about what files can be uploaded, that is irrelevant when uploading is not possible (which is the case with `readOnly`)